_posts/2020-12-19-generate-mini-site-with-metanorma-cli.adoc

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: "Generate mini site with Metanorma CLI"
+categories: about
+authors:
+  -
+    name: Abu Nashir
+    email: abunashir@gmail.com
+    social_links:
+      - https://github.com/abunashir
+
+excerpt: >-
+    Generate a collection mini site using Metanorma CLI
+---
+
+Each metanorma collection can be published as a mini site, there are couple of
+steps involed in this process to take a collection, compile it and generate the
+publishable site.
+
+To make this process easier, Metanorma has been offering a
+https://github.com/metanorma/mn-samples-itu/blob/master/Makefile[Makefile] that
+provides some targets to build the mini site. This has been working great until
+recently, but overtime it grew a lot and became harder to maintian, so we have
```

Comment:
maintian, attension, soruce: please check for typos.

Reply:
Thanks man, I will fix these